frontend/public/components/utils/headings.tsx

```diff
@@ -226,7 +226,10 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
         )}
         {helpText && (
           <Content>
-            <Content component={ContentVariants.p} className="pf-v6-u-mt-sm co-help-text">
+            <Content
+              component={ContentVariants.p}
+              className={classNames({ 'pf-v6-u-mt-sm': title }, 'pf-v6-u-text-color-subtle')}
```

Comment:
IMO we don't need the subtle text colouring for help text in the `PageHeading`. looking at the `react-component-groups` [equivalent component](https://github.com/patternfly/react-component-groups/blob/main/packages/module/src/PageHeader/PageHeader.tsx)

They don't change the colour of their `subtitle`, so I think we should match them and not recolour ours either

Reply:
Fair point.  Let me remove.

Reply:
[Updated](https://github.com/openshift/console/compare/8f527c12658661fcdf80162cca6a9b5082f66a16..634c84281bb7c2fa9e1dc5a01bae3038236cd539)